types/stylelint/index.d.ts

```diff
@@ -251,6 +252,22 @@ declare namespace stylelint {
 		fixable?: boolean;
 	};
 
+	export type Range = {
```

Comment:
[suggest] How about marking it as an unpublished type? Keeping published types minimal can make refactoring efficient later.

```suggestion
	/** @internal */
	export type Range = {
```